### PR TITLE
150 extend the copyright script for python files (backport dev-v10)

### LINKED ISSUE #150: Extend the copyright script for python files
#### Description
Currently the script `scripts/update_copyright.py` only changes `.cpp` and `.h` files. It should also consider python and shell files

#### Questions/Ideas
- The comment operator depends on the extension:
  - `.py`, `.sh` : `#`
  - `.cmd`, `.bat` : `REM`
  - `.h`, `.cpp`: `//`

### PR DESCRIPTION
**Pull request by folmos-at-orange**

### CODE REVIEW

File: test/LearningTest/MTdatasets/SpliceJunction/transcodeSpliceJunction.py [hunk 1]
```diff
@@ -1,37 +1,41 @@
...
-    fOutput.close()
-    fOutputData.close()
-
-
-Transcode("SpliceJunction.txt", "SpliceJunctionDNA.txt")
+# Copyright (c) 2024 Orange. All rights reserved.
+# This software is distributed under the BSD 3-Clause-clear License, the text of which is available
+# at https://spdx.org/licenses/BSD-3-Clause-Clear.html or see the "LICENSE" file for more details.
+
+import os
```
Comment (popescu-v) [inline test/LearningTest/MTdatasets/SpliceJunction/transcodeSpliceJunction.py:5]: Remove the trailing DOS-like carriage returns (^M) from the updates.

Comment (popescu-v) [inline test/LearningTest/MTdatasets/SpliceJunction/transcodeSpliceJunction.py:5]: Add to the commit message something like: "Also remove trailing DOS carriage returns where applicable"

**Review by popescu-v — APPROVED** [review]
LGTM with a small comment (commit message update - see comment).

File: test/LearningTest/MTdatasets/SpliceJunction/transcodeSpliceJunction.py (hunk 1 above)
Comment (folmos-at-orange) [reply test/LearningTest/MTdatasets/SpliceJunction/transcodeSpliceJunction.py:5]: Ok

**Pull request merged**